Project.toml

```diff
@@ -52,7 +52,7 @@ AtmosphericProfilesLibrary = "0.1"
 ClimaComms = "0.6.2"
 ClimaCore = "0.14.6"
 ClimaDiagnostics = "0.2"
-ClimaParams = "0.10.4"
+ClimaParams = "0.10.8"
```

Comment:
This is needed for the most recent version of RRTMGP

Reply:
If this is the case, it should be RRTMGP that defines this compat, not ClimaAtmos

Reply:
My mistake, the compat should change in Atmos. ClimaParams v0.10.8 is required to run radiation within Atmos, RRTMGP can run with older versions of CP.